docs/advanced/customization-guide.md

```diff
@@ -153,6 +153,16 @@ should be placed in a separate directory in order to avoid NFD unnecessarily
 trying to execute them. A subdirectory under the hooks directory can be used,
 for example `/etc/kubernetes/node-feature-discovery/source.d/conf/`.
 
+**NOTE:** Hooks are being DEPRECATED and will be removed in the future releases.
```

Comment:
For consistency with other changes, use "in a future release."

Reply:
done